# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/want-dynamic.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dietlibc-0.29.orig/dietfeatures.h	2005-09-24 12:28:43.704207728 -0600
+++ dietlibc-0.29/dietfeatures.h	2005-09-24 12:28:52.503869976 -0600
@@ -26,7 +26,7 @@
 #define WANT_THREAD_SAFE
 
 /* make the startcode, etc. dynamic aware ({con,de}structors) */
-/* #define WANT_DYNAMIC */
+#define WANT_DYNAMIC */
 
 /* GDB support in the dynamic linker */
 #define WANT_LD_SO_GDB_SUPPORT
